hail/python/hail/plot/plots.py

```diff
@@ -297,6 +297,9 @@ def manhattan(pvals, locus=None, title=None, size=4, hover_fields=None, collect_
         Whether to collect all values or downsample before plotting.
     n_divisions : int
         Factor by which to downsample (default value = 500). A lower input results in fewer output datapoints.
+    significance_line : float, optional
+        Location at which to plot a dotted red line for genome-wide
```

Comment:
This is not clear since the vertical axis is labeled by log transform of p-value. How about:
```
p-value at which to add a horizontal, dotted red line indicating genome-wide significance (e.g., 5e-8).  If ``None``, no line is added.
```